packages/superset-ui-chart/src/types/Column.ts

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable no-unused-vars */
+/* eslint-disable import/prefer-default-export */
```

Comment:
you can instead write `/* eslint-disable no-unused-vars, import/prefer-default-export */`